src/style.css

```
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  font-size: 100px;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  min-width: 300px;
+}
+
+.block--1 {
+  background-color: hsl(0, 100%, 50%);
+}
+
+.block--2 {
+  background-color: hsl(0, 100%, 40%);
+}
+
+.block--3 {
+  background-color: hsl(0, 100%, 30%);
+}
+
+.block--4 {
+  background-color: hsl(0, 100%, 20%);
+}
+
+.block--5 {
+  background-color: hsl(0, 100%, 10%);
+}
+
+.block--6 {
+  background-color: hsl(0, 100%, 0%);
+}
+
+@media (max-width: 599px) {
+  .block {
+    flex-basis: 100%;
+    order: 0;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
+  .block {
```

```suggestion
@media (min-width: 600px) {
  .block {
```
lets make it work like this